components/ui/ModalCreation.vue

```diff
@@ -1,31 +1,41 @@
 <template>
-  <Modal ref="modal" header="Create a project">
+  <Modal ref="modal" :header="formatMessage(messages.modalTitle)">
     <div class="modal-creation universal-labels">
       <div class="markdown-body">
-        <p>New projects are created as drafts and can be found under your profile page.</p>
+        <p>{{ formatMessage(messages.modalDescription) }}</p>
```

Comment:
Haven't reviewed the rest of this, but I do just want to say that there should probably be a note somewhere here when not in English that certain project metadata - like summaries and descriptions - must have an English translation

Reply:
I've added a note after the description, let me know if it suits you or not ^^.